Plugins~/Src/MeshSyncClientBlender/BlenderPyObjects/BlenderPyCommon.h

```diff
@@ -2,6 +2,7 @@
 
 #include <RNA_define.h> //PropertyRNA
 #include "MeshSync/msFoundation.h" //msPacked
+#include "msblenMacros.h"
 
```

Comment:
Is this import needed?

Reply:
Yes, it solves a compilation error with clang.